distributor-api-v1/operations.md

```diff
@@ -1055,9 +1053,9 @@ Gives a pricing information for the given configuration.
 
 |  | Property | Type | Description |
 | :--- | :--- | :--- | :--- |
-| `PaymentGatewayData` | string | required | Encoded credit card data obtained from the payment gateway specific library. More details [here](payment-gateway-data.md). |
-| `ObfuscatedCreditCardNumber` | string | required | Obfuscated credit card number, e.g.`411111******1111`. |
-| `Expiration` | string | required | Expiration of credit card in format `YYYY-MM`. Required with PCI Proxy payment gateway type. |
+| `PaymentGatewayData` | string | required | Encoded payment card data obtained from the payment gateway specific library. More details [here](payment-gateway-data.md). |
+| `Expiration` | string | required | Expiration of payment card in format `YYYY-MM`. |
```

Comment:
Is it correct that these are the field which API needs, the previous ones were outdated?

Reply:
in person discussion: yep